msal/src/main/java/com/microsoft/identity/client/PublicClientApplication.java

```diff
@@ -999,13 +1030,21 @@ private void loadMetaDataFromManifest() {
         mPublicClientConfiguration.mClientId = clientId;
     }
 
-    private void setupConfiguration(final int configResourceId, @NonNull final Context context) {
-        final PublicClientApplicationConfiguration developerConfig = loadConfiguration(context, configResourceId);
-        final PublicClientApplicationConfiguration defaultConfig = loadDefaultConfiguration(context);
-        defaultConfig.mergeConfiguration(developerConfig);
-        mPublicClientConfiguration = defaultConfig;
-        mPublicClientConfiguration.setAppContext(context);
-        mPublicClientConfiguration.setOAuth2TokenCache(getOAuth2TokenCache());
+    @VisibleForTesting
+    static PublicClientApplicationConfiguration loadConfiguration(@NonNull final Context context,
```

Comment:
Looks like this method could be refactored out.... it's only purpose now seems to be whether to determine the resource id is the resource id for the default configuration or not.  Since you've already gone most of the way... why not go all the way?

Reply:
No the method handle alternate resource configuration files that are bundled with the application. The boolean is just to keep the existing error message that is different whether it was using default or not.

Reply:
Ack.